sound/soc/intel/boards/sof_sdw_rt_sdca_jack_common.c

```diff
@@ -109,7 +109,7 @@ int rt_sdca_jack_rtd_init(struct snd_soc_pcm_runtime *rtd, struct snd_soc_dai *d
 		return -ENOMEM;
 
 	for (i = 0; i < ARRAY_SIZE(need_sdca_suffix); i++) {
-		if (strstr(codec_dai->name, need_sdca_suffix[i])) {
+		if (strstr(component->name_prefix, need_sdca_suffix[i])) {
```

Comment:
@ujfalusi  how is this even possible?

rt713 on MTL laptop:
codec_dai->name        : rt712-sdca-aif1

??

rt713 and rt712 are different parts.

Reply:
@bardliao might know, I can only say what I see.